/**/*:
  ros__parameters:
    storage:
      url: "http://localhost:8383"
      api_token: "access_token"
      bucket: "ros_data"
    pipelines:
      telemetry:
        filename_mode: "incremental"
        include_topics:
          - /recorder/input
        static_labels:
          source: telemetry
          robot: alpha
        split:
          max_duration_s: 10
          max_size_bytes: "5MB"
      rosout:
        filename_mode: "incremental"
        include_topics:
          - /rosout
        split:
          max_duration_s: 10
          max_size_bytes: "5MB"
      
